automate-project.py

```diff
@@ -91,7 +85,7 @@ def CreateGitHubRepo():
     GetCredentials()
     try:
         user = Github(username, password).get_user()
-        user.create_repo(repoName)
+        # user.create_repo(repoName)
```

Comment:
Are you sure this should be a comment?

Reply:
Oh i forgot to uncomment that while debugging. Will fix.